examples/web_export.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from myterial import orange
+from rich import print
+
+from brainrender import Scene
+
+print(f"[{orange}]Running example: {Path(__file__).name}")
+
+# Create a brainrender scene
+scene = Scene(title="brainrender web export")
+
+# Add brain regions
+scene.add_brain_region("MOs", "CA1", alpha=0.2, color="green")
```

Comment:
Should we add some asymmetry for easy visual inspection of potential future rotation bugs?

```suggestion
scene.add_brain_region("MOs", "CA1", alpha=0.2, color="green", hemisphere="right")
```

Reply:
Yes, great idea!